python/plugins/processing/gui/BatchOutputSelectionPanel.py

```diff
@@ -188,3 +218,21 @@ def setValue(self, text):
 
     def getValue(self):
         return str(self.text.text())
+
+    def outputName(self):
+        return self.text.text()
+
+    def isTemporaryOutput(self):
+        return self.actionTemporaryOutput.isChecked()
+
+    def onTemporaryOutputChanged(self) -> None:
```

Comment:
can you add typehints for these?

Reply:
Added. Should I also add typehints for the original functions in those classes that I edited? Might be useful in the future for maintenance of this code ...

Reply:
Definitely! 💯